dev/com.ibm.ws.bnd.annotations/bnd.bnd

```diff
@@ -24,5 +24,5 @@ Private-Package: com.ibm.ws.bnd.metatype.annotation, \
 publish.wlp.jar.disabled: true
 
 -buildpath: \
-	biz.aQute.bnd:biz.aQute.bnd.annotation;version=3.3.0, \
+	biz.aQute.bnd.annotation;version=3.3.0, \
```

Comment:
Are you sure about this? The original co-ordinates appear to match up with what is in artifactory...
https://na.artifactory.swg-devops.com/artifactory/webapp/#/artifacts/browse/tree/General/wasliberty-open-liberty/biz/aQute/bnd/biz.aQute.bnd.annotation/3.3.0/biz.aQute.bnd.annotation-3.3.0.jar

There is a 2.4.0 version in maven central that uses the new co-ordinates but I don't think you want that either.

Reply:
For `-buildpath`, each clause is a Bundle-Symbolic name (and other attributes like version). In this case `biz.aQute.bnd.annotation` is a bundle and thus has a Bundle-SymbolicName. For artifacts in a maven repo which do not have a Bundle-SymbolicName, Bnd allows them to be located using the synthetic Bundle-SymbolicName of `groupId:artifactId`.  

But since `biz.aQute.bnd.annotation` is a bundle and thus has a Bundle-SymbolicName, you cannot locate it with `biz.aQute.bnd:biz.aQute.bnd.annotation` since that is not its Bundle-SymbolicName.